client/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Graham's Search</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="shortcut icon" href="./img/favicon.ico" type="image/x-icon" />
+  </head>
+  <body>
+    <main class="wrapper">
+      <h1 class="title">Graham</h1>
+      <form>
+        <input
+          type="text"
+          class="form_child"
+          id="search_val"
+          placeholder="Search Graham"
+        />
+        <img class="mag_logo" src="./img/search.png" />
```

Comment:
Just curious why mag_logo

Reply:
mag was short for magnifying glass, maybe i could have named it better

Reply:
Just so you / others won't get confused or you can note in README